internal/dcache/clustermap/clustermap.go

```diff
@@ -449,6 +459,46 @@ func (c *ClusterMap) getOfflineMVs() map[string]dcache.MirroredVolume {
 	return offlineMVs
 }
 
+func (c *ClusterMap) getActiveMVsForRV(rvName string) map[string]struct{} {
+	//
+	// Save a copy of the clusterMap pointer to use for accessing RVMap and MVMap, so that both
+	// correspond to the same instance of clusterMap.
+	//
+	localMap := c.getLocalMap()
+
+	rv, ok := localMap.RVMap[rvName]
+	if !ok {
+		common.Assert(false, rvName)
+		return nil
+	}
+
+	rvIsOffline := (rv.State == dcache.StateOffline)
+
+	activeMVs := make(map[string]struct{})
```

Comment:
If RV is offline in the clustermap, we can directly return nil here so that we don't loop over the MVs below.